### DISCUSSION TITLE
Alteration "confusion"

### DISCUSSION THREAD

**Discussion by KirIcare** [Crownicles/Crownicles#1913] [Nouvelles idées]
Créer une altération d'état "confusion"
Une fois affecté, on peux continuer à jouer normalement à l'exception d'un détail: après un évent, lors du choix de la destination, seul 1️⃣ et 2️⃣ sont affiché, pas les noms de destination, ni l'émoji, ni le temps de trajet.
L'altération aurais une durée en évent et non pas en heure (5 ou 6 events me paraît bien)

**Comment by QQtin3** [comment]
Très honnêtement je trouve l'idée super sympa et drôle, c'est pas une altération "chiante" et qui t'empêche de jouer, juste qui t'embête un peu

**Comment by BastLast** [comment]
Le code des altérations d'état est vraiment pas prévu pour ce genre de mécaniques 😭

**Reply by KirIcare** [reply]
On est pas obligé d'appeler ça "une altération" dans ce cas 😅

**Reply by BastLast** [reply]
> On est pas obligé d'appeler ça "une altération" dans ce cas 😅

Ca le rend pas plus simple à coder mdr
